lua/typescript-tools/custom-commands/batch_code_fixes.lua

```diff
@@ -0,0 +1,81 @@
+local c = require "typescript-tools.protocol.constants"
```

Comment:
I'm wondering, maybe we can promote this to it's own custom lsp method then all language feature handlers will be in one place. Batching multiple requests in one lsp request is done for e.g. in code actions: https://github.com/pmizio/typescript-tools.nvim/blob/c5a11f8aee05a4790bf16cf198a0d7cf24a76b25/lua/typescript-tools/protocol/text_document/code_action/init.lua#L35

Reply:
done